config/sem_dedup_config.yaml

```diff
@@ -0,0 +1,30 @@
+# Configuration file for semdantic dedup
+cache_dir: "semdedup_cache"
+num_files: 16
+id_col_name: "id"
+id_col_type: "int"
+input_column: "text"
+
+# Embeddings configuration
+embeddings_save_loc: "embeddings"
+embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
+embedding_batch_size: 128
+embedding_max_mem_gb: 25
+
+# Clustering configuration
+clustering_save_loc: "clustering_results"
+n_clusters: 1000
+seed: 1234
+max_iter: 100
+Kmeans_with_cos_dist: false
+
+# Semdedup configuration
+which_to_keep: "hard"
+largest_cluster_size_to_process: 100000
+sim_metric: "cosine"
+
+# Extract dedup configuration
+eps_thresholds: "0.01 0.001"
```

Comment:
Can this be changed to a list instead of this string?
Something like
```yaml
eps_thresholds:
- 0.01
- 0.001
```
You'll need to change the parsing logic and docstring too obviously

Reply:
Resolved by https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/16ad76073206aedb5c96b6771ca0f56a99c321f2